src/module/functions/mod.rs

```diff
@@ -119,6 +128,15 @@ impl FunctionKind {
             _ => panic!("not a local function"),
         }
     }
+
+    /// Get the underlying `FunctionKind::Uninitialized` or panic if this is
+    /// not an uninitialized function
+    pub fn unwrap_uninitialized(&self) -> &TypeId {
```

Comment:
Hm, I actually think we shouldn't expose this method, since the uninitialized variant is sort of semi-private, and you can't really do anything with it anyways.

Let's remove this method from the PR. Sorry for not catching this earlier!

Reply:
@fitzgen done. I have updated the PR.